packages/zoe/NEWS.md

```diff
@@ -1,4 +1,90 @@
 User-visible changes in @agoric/zoe:
+
+## Release 0.3.0 (?)
+
+The structure of offer rules has changed to:
+
+```js
+{
+  offer: { Asset: moola(4 )},
```

Comment:
We should explain that moola(4) and simoleans(15) are helpers that create ERTP amounts.